conu/utils/core.py

```diff
@@ -6,25 +6,27 @@
 import logging
 import subprocess
 import os
-import tempfile
 import shutil
 import socket
 
+
 logger = logging.getLogger(__name__)
 
 
-# TODO: make this function to be less magic and use it everywhere
-def run_cmd(cmd, raw=False, **kwargs):
-    if not raw:
-        logger.debug("command: %s" % cmd)
-        if isinstance(cmd, str):
-            output = subprocess.check_output(cmd.split(" "), **kwargs)
-        else:
-            output = subprocess.check_output(cmd, **kwargs)
+# TODO: utilize this function everywhere and replace subprocess usage
+def run_cmd(cmd, return_output=False):
+    """
+    run provided command, raise subprocess.CalledProcessError if it fails
```

Comment:
Here it could be emphasized that it is being run on host and under which user.

Reply:
will do